resources.md

```diff
@@ -216,6 +216,24 @@ management, security configuration, and much more. Watch one
 of the demo videos and find details in the [comprehensive user
 documentation](https://docs.starburst.io/starburst-galaxy/index.html).
 
+</div>
+
+<div markdown="1" class="item">
+
+#### [Varada Control Center](https://varada.io/vcc/)
+
+Varada is the only solution that provides dynamic big data indexing for Trino,
```

Comment:
Reword .. since we know there are other solutions that... 

Varada provides dynamic


Also do that change in the other section